### PR TITLE
fix: deprecate updating any member

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR:
- deprecate updating any member

### CODE REVIEW

File: src/member/api.ts [hunk 1]
```diff
@@ -97,37 +98,50 @@ export const getMemberStorageFiles = async (
...
+    if (enableSaveActions !== undefined) {
+      body.enableSaveActions = enableSaveActions;
+    }
+    if (Object.keys(body).length) {
+      return axios
+        .patch<CompleteMember>(url.toString(), body)
+        .then(({ data }) => data);
+    } else {
+      return null;
+    }
```
Comment (ReidyT) [inline src/member/api.ts:134]: It's more a question, but I'm wondering if these checks are necessary here? Aren't we already checking in the frontend and in the backend? Also, what is happening in the frontend when returning null?

What do you think @pyphilia?

**Review by ReidyT — DISMISSED** [review]
Thanks for this PR, it looks good to me 😄. I just let one question.

File: src/hooks/mention.test.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ describe('Chat Mention Hooks', () => {
   });
 
   describe('useMentions', () => {
-    const currentMemberRoute = `/${buildGetCurrentMemberRoute()}`;
+    const currentMemberRoute = buildGetCurrentMemberRoute();
```
Comment (pyphilia) [inline src/hooks/mention.test.ts:21]: Not against, but maybe we want to do this together with all the other routes. Should we do it in another PR?

File: src/member/api.ts (hunk 1 above)
Comment (pyphilia) [reply src/member/api.ts:134]: I think it's okay to do those checks here. Maybe we want sth global so we indeed don't do the same checks manually everywhere? For now I would let it as done.
However you should maybe throw if the body is empty instead of returning null.

**Review by pyphilia — CHANGES_REQUESTED** [review]

**Comment by spaenleh** [comment]
@pyphilia @ReidyT I have updated this PR to only deprecate the editMember hook. The other changes will be implemented in a different PR

**Review by pyphilia — APPROVED** [review]
LGTM 🚒
